src/main/java/com/atlassian/bitbucket/jenkins/internal/util/SystemPropertiesConstants.java

```diff
@@ -0,0 +1,47 @@
+package com.atlassian.bitbucket.jenkins.internal.util;
+
+/**
+ * A list of all system properties used by the plugin.
+ */
+public class SystemPropertiesConstants {
+
+    /**
+     * If set to true, this property will prevent build statuses being sent by jobs to Bitbucket. This applies to all
+     * jobs running on the instance. This may be done for testing purposes, or to stop multiple Jenkins instances posting
+     * duplicate build statuses to Bitbucket.
+     * Defaults to FALSE.
+     */
+    public static final String BUILD_STATUS_DISABLED_KEY = "bitbucket.status.disable";
```

Comment:
I wonder if we should invert this, it is hard to wrap your mind around the double negative. If we made this 
`BUILD_STATUS_ENABLED_KEY = "bitbucket.status.enabled"` and default to `true` it makes more logical sense I think.

Reply:
Disabled = false was a conscious decision. I felt having the option read "build status enabled" makes it feel like both options are equally weighted, you might choose one or the other which I don't think is correct. "Build status disabled" feels you're overriding some feature of how the program works, which feels more correct as to what this is doing.
That said I don't feel strongly. I'm happy to change it if you do.

Reply:
Na, your reason makes perfect sense. I'm just wary of double negatives, but I think you're right. Let's leave it